hexrd/imageseries/load/imagefiles.py

```diff
@@ -64,6 +67,8 @@ def __getitem__(self, key):
             if np.max(data) > dinfo.max:
                 raise RuntimeError("specified dtype will truncate image")
             return np.array(data, dtype=self._dtype)
+        elif file_extension == '.gel':
+            return data.astype(np.float64)**2 * GEL_SCALE_FACTOR
```

Comment:
It looks like the gel processing should go before the dtype check, not as an elif.  If the dtype is not None, then it returns the image, so it will usually never get to the check for gel data.  In fact, we can probably eliminate the whole dtype check.  That is checking whether the data max is larger than the max for that dtype, and I don't see how that can happen if the image is that dtype.  Also, it wastes time since it has to process the entire image to find the max. 
 Hard to understand my own code! In any case, the gel data processing  should come before that check if we keep it.